Quadriga/src/main/java/edu/asu/spring/quadriga/web/resolver/AddResolverController.java

```diff
@@ -50,4 +58,26 @@ public String addResolver(Principal principal,
 
         return "redirect:/auth/resolvers";
     }
+
+    @RequestMapping(method = RequestMethod.POST, value = "auth/resolvers/testAdd")
+    public @ResponseBody ResponseEntity<String> testAddPageSuccess(@RequestParam("data") JSONObject data)
```

Comment:
since you're returning responseEntity, you don't need the responsebody annotation if I'm not wrong.